### DISCUSSION TITLE
Fazer os cursos com o VS Community no lugar do Code

### DISCUSSION THREAD

**Discussion by RochaRaphael** [balta-io/forum#52] [Tira Dúvidas] [answered]
Olá, gostaria de saber se terei muitos problemas em fazer os cursos com o VS Community, pois é o que tenho mais familiaridade, ou se realmente é melhor eu fazer com o VS code?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @RochaRaphael, como vai?

Embora o VS seja uma IDE e o Code apenas um editor de código, você pode realizar os cursos no VS normalmente.
Particularmente, recomendo começar pelo Code por que ele te ajuda menos, te força escrever mais, melhor e pensar mais no que está fazendo.

A partir do curso de ASP.NET é com Visual Studio e Rider.
